targets/TARGET_STM/TARGET_STM32L4/TARGET_STM32L496xG/device/TOOLCHAIN_IAR/stm32l496xx.icf

```diff
@@ -1,7 +1,10 @@
+if (!isdefinedsymbol(MBED_APP_START)) { define symbol MBED_APP_START = 0x08000000; }
+if (!isdefinedsymbol(MBED_APP_SIZE)) { define symbol MBED_APP_SIZE = 0x80000; }

```

Comment:
Not 0x100000 ?

Reply:
That's the previous value, maybe it's a other issue - but not familiar enough with details
The change only not to be hardcoded

Reply:
> That's the previous value

No... it was 0x100000 
define symbol __region_ROM_end__   = 0x08000000 + 0x100000 - 1;

Reply:
you are right, checking
maybe i took it from other target ( 429)